pages.fr/common/acyclic.md

```diff
@@ -0,0 +1,17 @@
+# acyclic
+
+> Construit un graphe orienté acyclique en inversant quelques sommets.
+> Filtres Graphviz : `acyclic`, `bcomps`, `comps`, `edgepaint`, `gvcolor`, `gvpack`, `mingle`, `nop`, `sccmap`, `tred`, & `unflatten`.
+> Plus d'informations : <https://graphviz.org/pdf/acyclic.1.pdf>.
```

Comment:
```suggestion
> Plus d'informations : <https://graphviz.org/pdf/acyclic.1.pdf>.
```
We agreed on a regular space, right? Did you copy this from somewhere?

Reply:
No, we agreed to keep the correct non-breaking space but we accept the regular space, and later we can run a tool and standardize the space.